rts/at/util/src/main/java/org/jboss/jbossts/star/util/exception/GenericATException.java

```diff
@@ -0,0 +1,31 @@
+package org.jboss.jbossts.star.util.exception;
+
+import javax.ws.rs.WebApplicationException;
+import java.net.URL;
+public class GenericATException extends WebApplicationException {
```

Comment:
I don't understand what's the purpose of creating the `GenericATException`. I can't find the usage in the code. Is it meant as some SDK?
Can you please elaborate what is that meant for? Thanks.

Reply:
Removed as no use right now